global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -415,10 +509,25 @@ data:
               text: {{"'{{template \"slack.sapcc.silence1Month.actionText\" . }}'"}}
               value: {{"'{{template \"slack.sapcc.silence1Month.actionValue\" . }}'"}}
 
-    - name: slack_vmware_warning
+    # slack duty api
+    - name: slack_api_info
       slack_configs:
-        - channel: '#alert-{{"{{ .CommonLabels.tier }}"}}-{{"{{ .CommonLabels.severity }}"}}'
-          api_url: {{ required "slack.vmware_warning_webhook_url undefined" .Values.slack.vmware_warning_webhook_url | quote }}
+        - channel: '#alert-api-{{"{{ .CommonLabels.severity }}"}}'
```

Comment:
Please be consistent. Either hardcoded severity or dynamic.

Reply:
but here was a benefit of doing it dynamic, bcz. the severity is limited by the route definition and is more or less constant. The more dynamic tier/service thing was to agil for my taste and created messages to non existing channels

Reply:
I thinks it's confusing to hardcode the name of the receiver to `slack_api_info` but have it dynamic in the actual configuration`channel: '#alert-api-{{"{{ .CommonLabels.severity }}"}}'`. 
So, given the name, this receiver is meant to only handle api info alerts, no?

Reply:
i mixed it up - you're completely right //sorry